db/models/song.js

```diff
@@ -0,0 +1,69 @@
+const Sequelize = require("sequelize");
+const db = require("../db");
+const Song = db.define(
+  "song",
+  {
+    title: {
+      type: Sequelize.STRING,
+      unique: false,
+      allowNull: false,
+    },
+    duration: {
+      type: Sequelize.FLOAT,
+      allowNull: false,
+      validate: {
+        isFloat: true,
+        min: 0.0,
+        max: 1800.0,
+      },
+    },
+    fileKey: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    fileURL: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      validate: {
+        isUrl: true,
+      },
+    },
+    genre: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    photoUrl: {
```

Comment:
Small typo, the schema has it as "photoURL"

Reply:
done